backend/modules/parsers/multimodalparser.py

```diff
@@ -153,38 +144,57 @@ async def get_chunks(
         """
         final_texts = []
         try:
-            if not filepath.endswith(".pdf"):
+            if (
+                not filepath.endswith(".pdf")
+                and not filepath.endswith(".png")
+                and not filepath.endswith(".jpeg")
+                and not filepath.endswith(".jpg")
+            ):
                 logger.error(
-                    "Invalid file extension. MultiModalParser only supports PDF files."
+                    "Invalid file extension. MultiModalParser only supports PDF, PNG, JPEG, JPG files."
                 )
                 return []
```

Comment:
Again, in my opinion, parsers should raise exceptions and callers should decide what they want to do with the error